code/ui/manager/src/components/notifications/NotificationItem.tsx

```diff
@@ -76,20 +77,9 @@ const ItemContent: FC<Pick<State['notifications'][0], 'icon' | 'content'>> = ({
   icon,
   content: { headline, subHeadline },
 }) => {
-  const theme = useTheme();
-  const defaultColor = theme.base === 'dark' ? theme.color.mediumdark : theme.color.mediumlight;
   return (
     <>
-      {!icon || (
-        <NotificationIconWrapper>
-          <Icons
-            icon={icon.name as IconsProps['icon']}
-            width={16}
-            height={16}
-            color={icon.color || defaultColor}
-          />
-        </NotificationIconWrapper>
-      )}
```

Comment:
This is a breaking change.

I think it's also inconsistent with the types definition for `addNotification`:
https://github.com/storybookjs/storybook/blob/eb6d200a6a0fa9fc4775b267f433704e1c578078/code/lib/types/src/modules/api.ts#L123-L135

Reply:
I updated the version to make it backward compatible. I'm checking if this is a valid react component. If it's not, then I'm not showing it.